rectype,issueid,project_owner,project_name,actor,time,text,action,title,clean_text,Test
rectype,issueid,project_owner,project_name,actor,time,text,action,title,text,N
issue_title,44,pycqa,astroid,pylint-bot,2014-07-31 05:43:38,"Originally reported by: **BitBucket: [idella5](http://bitbucket.org/idella5)**

---

``` python
...S..F..
======================================================================
FAIL: test_pylint_config_attr (unittest_regrtest.NonRegressionTC)
----------------------------------------------------------------------
Traceback (most recent call last)
  File ""/mnt/gen2/TmpDir/portage/dev-python/astroid-1.2.0/work/astroid-1.2.0-python3_4/build/lib/astroid/test/unittest_regrtest.py"", line 116, in test_pylint_config_attr
    expect)
  File ""/usr/lib64/python3.4/unittest/case.py"", line 992, in assertListEqual
    self.assertSequenceEqual(list1, list2, msg, seq_type=list)
  File ""/usr/lib64/python3.4/unittest/case.py"", line 974, in assertSequenceEqual
    self.fail(msg)
  File ""/usr/lib64/python3.4/unittest/case.py"", line 639, in fail
    raise self.failureException(msg)
AssertionError: Lists differ: ['Opt[84 chars]enChecker', 'BaseChecker', 'OptionsProviderMixIn', 'ASTWalker'] != ['Opt[84 chars]enChecker', 'BaseChecker', 'OptionsProviderMixIn']

First list contains 1 additional elements.
First extra element 7:
ASTWalker

  ['OptionsManagerMixIn',
   'object',
   'MessagesHandlerMixIn',
   'ReportsHandlerMixIn',
   'BaseTokenChecker',
   'BaseChecker',
-  'OptionsProviderMixIn',
?                        ^

+  'OptionsProviderMixIn']
?                        ^

-  'ASTWalker']

                              no stdout                               
                              no stderr                               
======================================================================

Ran 343 test cases in 6.75s (6.75s CPU), 1 failures, 17 skipped
12 modules OK (1 failed)
```

Looks fairly straight forward.  Last char a ',' in 1, ']"" in the other.
ditto python3.3.  Can you replicate? Do you require anything further?

---
- Bitbucket: https://bitbucket.org/logilab/astroid/issue/44
",start issue,test_pylint_config_attr fails in version 1.2.0 under py3,origin report bitbucket idella5 look fairli straight forward last char 1 ditto python33 can replic Do requir anyth bitbucket,N
issue_closed,44,pycqa,astroid,pylint-bot,2015-12-09 05:52:19,nan,closed issue,test_pylint_config_attr fails in version 1.2.0 under py3,nan,N
issue_comment,44,pycqa,astroid,pylint-bot,2014-07-31 05:50:15,"_Original comment by_ **Sylvain Thénault (BitBucket: [sthenault](http://bitbucket.org/sthenault), GitHub: @sthenault?)**:

---

This looks like a version mismatch between astroid and pylint. Are you sure
 to use pylint 1.3 together with astroid 1.2?
",nan,nan,origin comment sylvain thénault bitbucket sthenault github sthenault thi look like version mismatch astroid pylint are sure use pylint 13 togeth astroid 12,N
issue_comment,44,pycqa,astroid,pylint-bot,2014-07-31 06:28:08,"_Original comment by_ **BitBucket: [idella5](http://bitbucket.org/idella5)**:

---

ok fine.  astroid 1.2 requires pylint 1.3. 

---

Ran 343 test cases in 6.74s (6.74s CPU), 17 skipped
All 13 modules OK

Feel free to close
",nan,nan,origin comment bitbucket idella5 ok fine astroid 12 requir pylint 13 ran 343 test case 674 674 cpu 17 skip all 13 modul OK feel free close,Y
issue_comment,44,pycqa,astroid,pylint-bot,2014-07-31 06:30:44,"_Original comment by_ **Sylvain Thénault (BitBucket: [sthenault](http://bitbucket.org/sthenault), GitHub: @sthenault?)**:

---

pylint/astroid version mismatch
",nan,nan,origin comment sylvain thénault bitbucket sthenault github sthenault pylintastroid version mismatch,N
